include/xplat/WinObjCRTExport.h

```diff
@@ -15,6 +15,9 @@
 //******************************************************************************
 #pragma once
 
+// OSX has import/export on by default, making these unnecessary
```

Comment:
we still need to be at least extern C, my bad

---
In reply to: [72322401](https://github.com/Microsoft/WinObjC/pull/634#discussion_r72322401) [](ancestors = 72322401)